.github/workflows/import_pending_sql.yml

```diff
@@ -27,8 +29,10 @@ jobs:
           git commit -am "chore(DB): import pending SQL update file" -m "Referenced commit(s): ${GITHUB_SHA}" || true
         env:
           BRANCH: ${{ steps.extract_branch.outputs.branch }}
+        if: github.repository == 'azerothcore/azerothcore-wotlk'
       - name: Push changes
         uses: ad-m/github-push-action@fe38f0a751bf9149f0270cc1fe20bf9156854365
+        if: github.repository == 'azerothcore/azerothcore-wotlk'
```

Comment:
is there a way to make this `if` kind of "global" and apply it to the whole `import-pending-sql` step instead of adding it manually multiple times?

anyway, this is good! we needed it

Reply:
I was doing some more checking on the github docs, might be able to do it at the jobs level instead of the steps level. I will do some more testing.

Reply:
@FrancescoBorzi moving the if to the jobs level worked:

![image](https://user-images.githubusercontent.com/4015312/92334627-a67eed80-f044-11ea-9e62-1b0d36db540f.png)